src/seedu/addressbook/commands/ListCommand.java

```diff
@@ -20,6 +21,7 @@
     @Override
     public CommandResult execute() {
         List<ReadOnlyPerson> allPersons = addressBook.getAllPersons().immutableListView();
+
```

Comment:
Sorry, what's the change in this file? (Just making sure it's as clean a pull request as possible)